src/styles/Card.js

```diff
@@ -62,4 +62,22 @@ const CardPadding = styled.div`
   padding: ${size.small} ${size.tiny} 0;
 `;
 
-export {Card, CardPadding, FittedCard};
+const ContextStyle = styled(Card)`
+  #context-div {
```

Comment:
This can be a classname. I have a preference for using id's for JavaScript manipulations.